go.mod

```diff
@@ -13,6 +13,8 @@ require (
 	google.golang.org/protobuf v1.28.1
 )
 
+require github.com/mattn/go-sqlite3 v1.14.15 // indirect
+
```

Comment:
Another nitpick from me; I'm not sure why this is added _here_, and it should be a direct dependency. I'd just add it to the list above and remove the `// indirect` comment (or just run `go mod tidy`, which also will mark it as a direct dependency). I noticed that this happens when using `go get`, so I guess it is fine, just kinda weird. But it should be a direct dependency anyways.

Reply:
Imo the entire go.mod and go.sum thing is a bit of a missfeature the way it works now since its main function is to cause irrelevant merge conflicts in files that are by and large computer generated and maintained, so just run go mod tidy and be done with it I suppose... Maybe we should just do 'go mod tidy' in make check to see if it generates a diff....